lib/ain-grpc/src/lib.rs

```diff
@@ -57,12 +56,14 @@ pub fn init_logging() {
 
 pub fn init_services() {
     info!("Init rs services");
+    IS_INITIALIZED.store(true, Ordering::Relaxed);
     let _ = &*SERVICES;
 }
 
 pub fn init_network_services(json_addr: &str, grpc_addr: &str) -> Result<()> {
     init_network_json_rpc_service(&SERVICES, json_addr)?;
     init_network_grpc_service(&SERVICES, grpc_addr)?;
+    IS_INITIALIZED.store(true, Ordering::Relaxed);
```

Comment:
Perhaps just init in one place, either here or after services // or perhaps granular flags.

Reply:
Wanted to cover all usage of SERVICES that would initialize it just to be safe. Another option was to assert it's initialization before calling `the init_network_*`